webapp/components/User/User.vue

```diff
@@ -1,32 +1,37 @@
 <template>
-  <div class="user" v-if="displayAnonymous">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:23:41Z" title="Wednesday, January 8th 2020, 4:23:41 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="user" v-if="displayAnonymous">
```


@mattwr18 do you think it would be a good idea to manage the `anonymous` state directly in the `Avatar` component? I feel it would make sense but haven't thought it through in detail.